variables.tf

```diff
@@ -158,6 +158,18 @@ variable "install_metrics_server" {
   default     = false
 }
 
+variable "storage_bucket_versioning" {
+  description = "Enable bucket versioning (Disabled by default due to deletion timeouts)."
```

Comment:
Should this have "When enabled, set `storage_bucket_version_ttl`."  or something?

Reply:
I've just set a default of 7 days for versioning.